### PR TITLE
chore: explicitly declare the required node version and add .nvmrc

### PR DESCRIPTION
**Pull request by Balastrong**
## The problem

When running yarn the first time I got an error because my local node version didn't match a dependency's requirement (`semantic-release@20.1.0` requires node `>=18`)

```
yarn install v1.22.19
[1/4] 🔍  Resolving packages...
[2/4] 🚚  Fetching packages...
error semantic-release@20.1.0: The engine "node" is incompatible with this module. Expected version ">=18". Got "16.15.1"
error Found incompatible module.
```

## Proposed solution

I explicitly declared that this project requires node >= 18 and added a `.nvmrc` to automatically switch the local node version for nvm users.

This way is much more contributor-friendly, no errors on console and the correct node version is automatically loaded :)

### CODE REVIEW

File: .nvmrc [hunk 1]
```diff
@@ -0,0 +1 @@
+v18
```
Comment (aiji42) [inline .nvmrc:1]: I do not use `nvm` and I am not smart enough to remember that when the lowest version of node required for this project changes in the future (when I change `engine.node` in package.json), I need to change `.nvmrc` at the same time.
Therefore, I do not want `.nvmrc`, `.node-version`, etc. to be included in the project.

**Review by aiji42 — CHANGES_REQUESTED** [review]

**Comment by aiji42** [comment]
@Balastrong Thank you!
Please check my review comment.

File: .nvmrc (hunk 1 above)
Comment (Balastrong) [reply .nvmrc:1]: Ok, I'm removing it then :)

**Review by aiji42 — APPROVED** [review]

**Comment by aiji42** [comment]
Thanks! I will merge this.

**Comment by Balastrong** [comment]
Thank you :)

**Pull request merged**